pkg/names/names.go

```diff
@@ -172,6 +172,19 @@ const (
 	NetworkTypeMigrationMTUReady string = "NetworkTypeMigrationMTUReady"
 )
 
+const (
+	// MachineConfigLabelRoleKey label key for machine config role.
+	MachineConfigLabelRoleKey = "machineconfiguration.openshift.io/role"
+)
+
+var (
+	// When user deploys their own machine config for installing and configuring specific version of libreswan, then
+	// corresponding master and worker role machine configs annotation must have `user-ipsec-machine-config: true`.
+	// When CNO finds machine configs with the annotation, then it skips rendering its own IPsec machine configs
+	// and reuse already deployed user machine configs for the ovn-ipsec-host daemonset.
+	UserDefinedIPsecMachineConfigAnnotation = map[string]string{"user-ipsec-machine-config": "true"}
+)
```

Comment:
Why didn't you move 	masterRoleMachineConfigLabel nad workerRoleMachineConfigLabel here as well.

And we can also make the trick of the method with UserDefinedIPsecMachineConfigAnnotation

Reply:
yes, done.